src/routes/contact/+page.svelte

```diff
@@ -0,0 +1,182 @@
+<script lang="ts">
+    import * as Menu from 'menu/contact';
+
+    import type { LoadContact } from 'types/loadTypes';
+
+    import { resetUnsavedChanges, setUnsavedChanges, unsavedChanges } from 'stores/saved';
+    import { trySaveDataAsync } from 'helper/trySaveData.js';
+    import { SaveMessageType } from 'types/saveMessageType';
+    import { z } from 'zod';
+
+    import PageWrapper from 'elements/section/pageWrapper.svelte';
+    import TextLine from 'elements/text/textLine.svelte';
+    import HeadlinePage from 'elements/text/headlinePage.svelte';
+    import Input from 'elements/input/input.svelte';
+    import DropDown from 'elements/input/dropDown.svelte';
+    import TextArea from 'elements/input/textArea.svelte';
+    import SectionDashboard from 'elements/section/sectionDashboard.svelte';
+    import Button from 'elements/input/button.svelte';
+    import SaveMessage from 'elements/text/saveMessage.svelte';
+    import MessageWrapper from 'elements/text/messageWrapper.svelte';
+    import UnsavedChangesPopup from 'elements/popups/unsavedChangesPopup.svelte';
+    import UnsavedChangesCallbackWrapper from 'elements/navigation/unsavedChangesCallbackWrapper.svelte';
+    import { onMount } from 'svelte';
+
+    enum State {
+        Default,
+        EmailSend,
```

Comment:
I think this should be in the past form.
```suggestion
        EmailSent,
```